mono/profiler/proflog.c

```diff
@@ -2945,9 +3509,17 @@ create_profiler (const char *filename)
 		if (!start_helper_thread (prof))
 			prof->command_port = 0;
 	}
+
+	if (do_coverage) {
```

Comment:
you should omit braces when the inner code is just a single statement (same throughout the PR)